### ISSUE TITLE
IndentationError: expected an indented block

### ISSUE THREAD

**Issue by me-benni** [85pando/mcStats#10]
When trying to execute there appears to be a bug?
```
$ python mcStats.py --help
  File "mcStats.py", line 127
    print singlefile, 'is not a file'
        ^
IndentationError: expected an indented block
```

Is this bug related to the python version? I'm running 'Python 2.7.10'
I would have tried to fix this, but I have no clue of python.

**Comment by Gjum** [comment]
This is indeed an error in the [code](https://github.com/85pando/mcStats/blob/master/mcStats.py#L127), the print statement should be indented with 4 spaces more. You can fix this yourself, but it also seems that there is another issue, because `singlefile is not a file` ;)

**Comment by me-benni** [comment]
thanks, that did the trick :)

**Comment by Gjum** [comment]
I think this should be kept open, as the source code is still wrong.

**Comment by me-benni** [comment]
you might be right. it's a quick fix

**Comment by 85pando** [comment]
Its actually not an error in the code.

Singlefile is just the string for a single files filename. I fixed this in 84aa788fe2ff13d67ff2e10c221a58b9a86dc3a3
